guide/C/ch_invest.xml

```diff
@@ -1605,7 +1605,7 @@ Income
             <row>
               <entry>Assets:Brokerage Account:Stock:AMZN</entry>
 
-              <entry>-100</entry>
+              <entry>&mdash;100</entry>
```

Comment:
Hi John, 
Thanks for reviewing my mods. Yes, I've several times wondered  if a minus sign is actually supposed to be there. I found some-one else had used &mdash; and thought that made it really clear. I'll have a look at how &minus; looks. There are no examples of that already in the help or guide.
How do you want me to handle all your requested changes? A commit for each one or only where you say they should be a separate commit? If multiple commits, do I use the commit message for this whole mod or a specific commit message for each?